packages/klevu-ui/src/components/klevu-chat-layout/klevu-chat-layout.css

```diff
@@ -6,10 +6,14 @@
   display: flex !important;
   flex-direction: column;
   max-height: 100vh;
+  --chat-height: var(--klevu-chat-height, 200px);
+  --chat-max-height: var(--klevu-chat-max-height, 100vh);
```

Comment:
I think we should default to --chat-height.
100vh is the requirement for product-query, we should pass that value from the product-query component